testsuite/run_sets/secondary.yml

```diff
@@ -23,6 +23,7 @@
 - features/secondary/srv_handle_software_channels_with_ISS_v2.feature
 - features/secondary/srv_handle_config_channels_with_ISS_v2.feature
 - features/secondary/srv_salt_git_pillar.feature
+- features/secondary/srv_password_restriction.feature
```

Comment:
There is any special reason for that test to don't be able to run in parallel?

Reply:
We are changing the password requirements. During this time creating a user with linux password will fail.